README.md

```diff
@@ -113,26 +113,31 @@ Example:
 
 ## Development
 
-### Bazel install
+### Dependencies
 
-We use [bazel](https://bazel.build/) to build, and are pinned to an old version
-of it in `package.json`. So unless you have that specific version of bazel installed,
-you should run run `yarn bazel` instead of `bazel` in any of the below commands.
+- nodejs. Install from your operating system's package manger, by following
+  instructions on https://nodejs.org/en/, or by using
+  [NVM](https://github.com/nvm-sh/nvm)
+- yarn. Install from your operating system's package manager or by following
+  [instructions on yarnpkg.com](https://yarnpkg.com/en/docs/install).
+- bazel. Install from your operating system's package manager or by [following
+  instructions here](https://docs.bazel.build/versions/master/install.html).
 
 ### One-time setup
 
-Run `bazel run @nodejs//:yarn --script_path=yarn_install.sh && ./yarn_install.sh`
-to install the dependencies.
+Run `bazel run @nodejs//:yarn` to install dependencies.
+
+### Bazel install
 
-> This avoids occupying the `bazel` server, so that `yarn` can call `bazel`
-> again.
-> Ideally we should just use `bazel-run.sh @nodejs//:yarn`, see
-> https://stackoverflow.com/questions/47082298/how-can-users-get-bazel-run-sh
+We use [bazel](https://bazel.build/) to build, and are pinned to a specific
+version of it in `package.json` for reproducible builds. The build rules check
+for a compatible version of bazel, so it is generally safe to use your local
+installed version. If in doubt, you can run `yarn bazel` instead of `bazel` in
```

Comment:
I discovered that 'yarn bazel' does the wrong thing in some situations, because yarn runs its tools with a new temp dir on each invocation which confuses the bazel cache.  I think instead you should recommend something like

"you can run `node_modules/.bin/bazel` instead"

Reply:
that was fixed by adding `--incompatible_strict_action_env` - `yarn bazel` is fine (Angular team does it that way)